src/index.js

```diff
@@ -208,6 +209,7 @@ function addMetadata({ types: t }) {
     visitor: {
       Program(babelPath, state) {
         const GET_TEST_METADATA = 'getTestMetadata';
+        const CONTEXT = 'QUnit.config.current.testEnvironment';
```

Comment:
could this ever resolve to undefined? what is the behavior if/when its not defined?

Reply:
@scalvert would you know off hand if 'QUnit.config.current.testEnvironment' could ever potentially resolve to undefined?

Re: behavior if undefined, I think the plugin can have access to this and can do a check on its value. If it's undefined I think we can set it to 'this', but that sort of negates what this changes is trying to address.

Another option is to set it to the returned value of `getContext()`. In that case we must import `getContext()` from test-helpers. We aren't using getContext() because, in the case where no 'hooks' is passed into module() and this plugin inserts hooks, we get a weakmap error. I don't know why getContext() is undefined in this case.

Reply:
Actually, I don't think (or I'm not sure how) we can check `Qunit...` or `getContext()` in this stage, where the plugin is processing transformations. I think we can only check their values during the test's actual runtime.